qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -7,4 +7,4 @@ login.auto=false
 headless=true
 db.host=openmrs
 webdriver.gecko.driver=firefoxdriver/linux/geckodriver
-includes.csrftoken=true
+includes.csrftoken=true
```

Comment:
any reason for this change?

Reply:
Is this not part of the master.https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties#L10

Reply:
any reason for removing the last empty line?

Reply:
@kdaud  where is the last empty line removed here.

Reply:
checkout master on your local repo and see the file within your IDE!

Reply:
isn't there an empty line at the EOF in master?